js-intl-polyfill/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>cgg-parent</artifactId>
+    <groupId>org.pentaho.ctools</groupId>
+    <version>9.3.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>cgg-js-intl-polyfill</artifactId>
+  <packaging>pom</packaging>
+
+  <properties>
+    <nodejs.version>v15.1.0</nodejs.version>
```

Comment:
do you need this specific version? it would be better to use a LTS version

Reply:
Good point.
NodeJS is used at build time only (there are no tests).
The version of NodeJS inherited via the parent POM is `v10.0.0`.
While Rollup itself would work with NodeJS `v10.12.0`, our `rollup.config.js` script uses functionality only available since `v12.2.0`.

Now why `v15.1.0`? It seems, ... that's the version I have installed on my machine :-/
And I ended up not revisiting this choice.

The latest LTS version is `v14.17.0`, so I'll change to that one.

Thanks!